CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to WildFly Projects
+
+Thanks for considering contributing to WildFly and the other affiliated projects affiliated with it in the WildFly Commonhaus Organization.
+
+We expect all contributors and users to follow our [Code of Conduct](https://github.com/wildfly/wildfly-governance/tree/main/CODE_OF_CONDUCT.md) when communicating through project channels.
+
+The specific details describing how to contribute to our projects vary based on the project. Information about contributing to the top-level projects in the WildFly Commonhaus Organization can be found here:
+
+### Ironjacamar
+
+TODO add link to Ironjacamar CONTRIBUTING.md
+
+### JBeret
+
+TODO add link to JBeret CONTRIBUTING.md
+
+### JBoss Parent
+
+TODO add link to JBoss Parent CONTRIBUTING.md
+
+### JBoss Threads
+
+TODO add link to JBoss Threads CONTRIBUTING.md
+
+### JBoss WS
+
+TODO add link to JBoss WS CONTRIBUTING.md
+
+### Narayana
+
+TODO add link to Narayana CONTRIBUTING.md
```

Comment:
@bstansberry FYI I raised https://github.com/bstansberry/wildfly-governance/pull/1 to suggest an update for this

Reply:
Thanks @tomjenkinson